ui/pages/Project/components/RnaSeqOutliers.jsx

```diff
@@ -51,20 +53,20 @@ class RnaSeqOutliersGraph extends React.PureComponent {
       .append('g')
 
     dataPoints.append('circle')
-      .attr('cx', d => x(d.zScore))
-      .attr('cy', d => y(d.pValue))
-      .attr('r', d => (d.deltaPsi === undefined ? 3 : r(Math.abs(d.deltaPsi))))
+      .attr('cx', d => x(d[xField]))
+      .attr('cy', d => y(d[yField]))
+      .attr('r', 3)
```

Comment:
what does 3 mean here?

Reply:
The size for the dot. The size was previously `d.deltaPsi === undefined ? 3 : r(Math.abs(d.deltaPsi)))` so for the cases where we had `deltaPsi` the size was variable based on the data and for the places we didn't it was always `3`. Now that there is no data where we want the dot size to be dynamic, its just always the default, or `3`